experiments/list_unsupported.sh

```diff
@@ -61,19 +61,33 @@ if ! (gnat --version | grep -q 'GNAT GPL 2016') ; then
    exit 5
 fi
 
+# Sane ADA_INCLUDE_PATH and GPR_PROJECT_PATH?
 GNAT2GOTO=`command -v gnat2goto`
 ADA_HOME=`command -v gnat`
-ADA_HOME="$(dirname "$ADA_HOME")/.."
+ADA_HOME=$(cd "$(dirname ${ADA_HOME})/.." 2>/dev/null && pwd)
```

Comment:
does curly bracket expansion work if there are spaces on the path?

Also, I'm not sure, but is the `cd` just there to check if the path exists? If so, why not just `[ -d ... ]`?

Reply:
No, this is for path canonicalisation - it's crude, but idiomatic way to do so - `cd` into the directory, then `pwd` from that directory - it turns something like `foo/..` into `foo`, or `foo/bar/../wibble` into `foo/wibble`.

Reply:
Can this be turned into a function in a later PR? This sounds like useful information that will be lost otherwise. If we had a function called `path_canon` or something it would be easier to understand and use.